stable/datadog/values.yaml

```diff
@@ -189,6 +189,29 @@ datadog:
   #
   containerLogsPath: /var/lib/docker/containers
 
+  ## @param customLogFilesEnabled - boolean - optional - default: false
+  ## Enable this to define host logs to mount into the container.
+  ## ref: https://docs.datadoghq.com/agent/logs/?tab=tailexistingfiles#custom-log-collection
+  #
+  # customLogFilesEnabled: false
+
+  ## @param customLogFiles - array - optional - default: []
+  ## Format a list of files from the host to mount into the container. Format
+  ## must follow the "Custom log collection" format for files. Wildcard files
+  ## are NOT accepted at this time. In order for datadog to actually read these
```

Comment:
Let's rephrase `In order for datadog to actually read these logs,` to a simpler `For these logs to be read,`